### ISSUE TITLE
External links

### ISSUE THREAD

**Issue by 2manyvcos** [interactivethings/catalog#369]
I would like to add external links to the navigation bar.

So why not allow something like this:

```jsx
{
  path: 'https://www.github.com/xyz',
  title: <span><Icon name="github" /> GitHub</span>,
  // eventually
  external: true,
}
```

**Note:** To clarify, it already works but prints errors in the developer console when not in production mode

**Comment by herrstucki** [comment]
Yeah, this seems like a good thing to have! External menu entries should be visible as such (probably through an icon).
